figma-shop/src/app/pages/product-details/product-details.component.html

```diff
@@ -1,14 +1,143 @@
+<div class="details-container">
 <div class="details">
   <div class="details__images">
+    <div class="details__image-content">
+      <span class="count">-36%</span>
+      <span class="count">Free shipping</span>
+    </div>
     <div class="details__main-image">
-      <img src="../../../assets/images/food-image.jpg" alt="product">
+      <img class="details__image" src="{{ product.imgUrl }}" alt="{{ product.title }}">
```

Comment:
Use property binding for attributes instead of interpolation.

Reply:
OK